### ISSUE TITLE
Skipping 1.8 altogether. 1.9 branch is being worked on. sigh...

### ISSUE THREAD

**Issue by purpleposeidon** [ganymedes01/TheErebus#499]

**Comment by purpleposeidon** [comment]
I miiight be willing to at least get it compiling, but I couldn't even start this for several months.

**Comment by ganymedes01** [comment]
There is a 1.8 branch that @vadis365 started working on. It's still quite broken from what I can remember.
Feel free to send PR to it whenever you can as I doubt anyone will be touching it anytime soon.

**Comment by vadis365** [comment]
Any help would be welcome - all I have is the dimension working at a basic level.
The portal block will transfer you to The Erebus which exists with biomes and world gen working for the basic chunks. Atm vanilla blocks are used as placeholders because nothing else was done by the people that promised to help. (ex team members who did sod all and broke things anyway).
I will be looking in to working on this next month probably.

**Comment by vadis365** [comment]
@ganymedes01, @purpleposeidon  Erebus really needs some new coders to help with this - if you know anyone or could put a shout out for some help it would be greatly appreciated. The mod is more popular than ever with the last version (at time of typing this) having over 112K dl's. I am just not capable time or coding wise to achieve this without help. ;)

**Comment by vadis365** [comment]
Started something with 1.9 - everything is broken again - joy
No point porting to 1.8 as everything breaks again in 1.9...

**Comment by vadis365** [comment]
Going to re-write all the world gen code from scratch and hopefully make it more sensible and coherent.
Will take me a while to get anything done as I have a bit to learn with all the 1.9 changes.
Once we have a dimension and teleporter working then basic blocks will be will be sorted to facilitate terrain gen and possibly some of the decoration. After blocks I will work on mobs. Will probably need help with TE's (will see what has changed nearer the time) and any complex items.

**Comment by vadis365** [comment]
OK, got further with the 1.9 port than I did with 1.8.
Some basic terrain blocks are in and a few little worldgen tests.
I have started working on the logs and saplings for trees. Leaves will follow, after which I can add the trees back to decoration and then work on other parts of the generation; giant flowers, double height plants and other plants etc. Other wooden blocks, planks, slabs, fences, gates, doors etc. will follow on after. Once the wood is done I will start adding back the ores and the generation for them. I should be able to get most of the basic blocks in and functioning over the next few weeks. The jsons are taking a while to make but meh. Items are probably going to be more of a pain as some of them do rather complicated things.
